rmw_zenoh_cpp/src/detail/rmw_wait_set_data.hpp

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DETAIL__RMW_WAIT_SET_DATA_HPP_
+#define DETAIL__RMW_WAIT_SET_DATA_HPP_
+
+#include <condition_variable>
+#include <mutex>
+
+#include "rmw/rmw.h"
+
+namespace rmw_zenoh_cpp
+{
+
```

Comment:
Could we add the text from your PR description as a docstring here? It will be very useful.

Reply:
Added in [b6074f4](https://github.com/ros2/rmw_zenoh/pull/211/commits/b6074f42a8534cc8683addbb13ca66c58e736d72)